irepela/homework_5/non_unique_elements.py

```diff
@@ -0,0 +1,18 @@
+def non_unique_elements(data: list[int]) ->  list[int]:
+    """
+        Filters unique elements
+
+        Args:
+            data (str): list of integers to filter unique elements
+
+        Returns:
+            list[int]: list of non unique elements
+    """
+    unique_list = []
+    for num in data:
+        if data.count(num) == 1:
+            unique_list.append(num)
```

Comment:
why you need to define unique list if you can at once define non unique list

if data.count(num) > 1

Reply:
you can solve a task with 1 liner within list comprehension